lib/screen-flash.coffee

```diff
@@ -0,0 +1,11 @@
+module.exports =
+  flash: (editorElement) ->
+    normalBG = editorElement.style.background
+    c = @getConfig "color"
+    editorElement.style.background = "rgb(#{c.red},#{c.green},#{c.blue})"
+    setTimeout ->
+      editorElement.style.background = normalBG
+    , @getConfig "duration"
+
+  getConfig: (config) ->
+    atom.config.get "activate-power-mode.screenFlash.#{config}"
```

Comment:
There are some cases when the background changes permanently. Also there is a good variation of this with a `filter: invert(100%)`.
Nevertheless I think for now we will add only the audio effect, I hope the plugin api will be ready soon, so effects (particles, screen and audio) will be interchangeable.